plutus-use-cases/test/Spec/Crowdfunding.hs

```diff
@@ -72,17 +75,10 @@ successfulCampaign = checkCFTrace scenario1 $ do
     con2 <- contrib w2 c 600
     con3 <- contrib w3 c 800
     updateAll'
-    setValidationData $ ValidationData $ plc PendingTx {
-        pendingTxCurrentInput = (PendingTxIn (PendingTxOutRef 100 1) (), 600),
-        pendingTxOtherInputs  = (PendingTxIn (PendingTxOutRef 200 1) (), 800):[],
-        pendingTxOutputs      = []::[PendingTxOut CampaignActor],
-        pendingTxForge        = 0,
-        pendingTxFee          = 0,
-        pendingTxBlockHeight  = 11
-        }
+    setValidationData $ ValidationData $(plutus [| $(pendingTxCrowdfunding) 11 600 (Just 800) |])
     collect w1 c [(con2, w2, 600), (con3, w3, 800)]
     updateAll'
-    mapM_ (uncurry assertOwnFundsEq) [(w2, 400), (w3, 200), (w1, 1400)]
+    mapM_ (uncurry assertOwnFundsEq) [(w2, startingBalance - 600), (w3, startingBalance - 800), (w1, 600 + 800)]
```

Comment:
Could `traverse_` be used here? I believe it is more general.

Reply:
Yes, that's right!